sdk/vc-data/src/data/employment/v1.ts

```diff
@@ -14,12 +15,24 @@ import {
 
 // Person Related
 
+export type SalaryV1 = CreateThing<
+  'Salary',
+  {
+    gross: ExpandThing<MonetaryAmountRV1>
+    net: ExpandThing<MonetaryAmountRV1>
+    frequency: 'Daily' | 'Weekly' | 'Monthly' | 'Hourly' | 'Annual'
+  }
+>
+
 type PersonEmployeeRoleEV1Mixin = CreateThing<
   'PersonEmployeeRoleE',
   {
-    reference?: CreateExpandedThing<'ContactPoint'>
+    reference?: MaybeArray<CreateExpandedThing<'ContactPoint'>>
     skills?: MaybeArray<CreateExpandedThing<'DefinedTerm'> | string>
     worksFor: ExpandThing<OrganizationEV1>
+    offerLetter?: CreateThing<'URL'>
```

Comment:
This should be `CreateExpandedThing<'URL'> | string`, otherwise you'd only be able to do:

```
offerLetter: {'@type': 'URL'}
```

without any other fields or just using the URL directly.

(same for `experianceLetter`)

Reply:
👍 
made the requested changes.